third-party-licenses.csv

```diff
@@ -1,19 +1,11 @@
 "module name","licenses","repository","licenseUrl","parents"
-"@docusaurus/core@2.0.0-beta.17","MIT","https://github.com/facebook/docusaurus","https://github.com/facebook/docusaurus/raw/master/LICENSE","website"
-"@docusaurus/preset-classic@2.0.0-beta.17","MIT","https://github.com/facebook/docusaurus","https://github.com/facebook/docusaurus/raw/master/LICENSE","website"
-"@mdx-js/react@1.6.22","MIT","https://github.com/mdx-js/mdx","https://github.com/mdx-js/mdx/raw/master/license","website"
 "@types/lodash@4.14.181","MIT","https://github.com/DefinitelyTyped/DefinitelyTyped","https://github.com/DefinitelyTyped/DefinitelyTyped/raw/master/LICENSE","cursorless"
 "ansicolor@1.1.100","Unlicense","https://github.com/xpl/ansicolor","https://github.com/xpl/ansicolor/raw/master/LICENSE","metals"
 "brace-expansion@2.0.1","MIT","https://github.com/juliangruber/brace-expansion","https://github.com/juliangruber/brace-expansion/raw/master/LICENSE","vscode-fileutils"
-"clsx@1.1.1","MIT","https://github.com/lukeed/clsx","https://github.com/lukeed/clsx/raw/master/license","website"
 "immutability-helper@3.1.1","MIT","https://github.com/kolodny/immutability-helper","https://github.com/kolodny/immutability-helper/raw/master/LICENSE","cursorless"
 "jsonc-parser@2.3.1","MIT","https://github.com/microsoft/node-jsonc-parser","https://github.com/microsoft/node-jsonc-parser/raw/master/LICENSE.md","parse-tree"
-"mdast-util-find-and-replace@2.1.0","MIT","https://github.com/syntax-tree/mdast-util-find-and-replace","https://github.com/syntax-tree/mdast-util-find-and-replace/raw/master/license","website"
+"lodash@4.17.21","MIT","https://github.com/lodash/lodash","https://github.com/lodash/lodash/raw/master/LICENSE","cursorless"
```

Comment:
Interesting that this list has changed, despite the affected packages not changing in package.json or yarn.lock.